src/components/Postiterie.vue

```diff
@@ -207,16 +207,20 @@ export default {
 
     sendFile: function(){
       let $this = this;
+      
       let fileContent = '<?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" /></root></mxGraphModel>'
```

Comment:
Finding this long string in the middle of you function is awkward, hide it somewhere? On its own file?

Reply:
This should be fixed by the other pr that deals with generating the xml